src/app/issure/dashboard/dashboard.component.html

```diff
@@ -80,12 +80,13 @@ <h6 class="modal-title primary-text" id="prewiewProfile">Edit issuer details</h6
                                 aria-hidden="true">&times;</span></button>
                     </div>
                     <div class="modal-body" *ngIf="fields">
-                        <form [formGroup]="form2" (ngSubmit)="submit()">
+                        <form [formGroup]="form2" (ngSubmit)="submit()" id="myForm">
                             <formly-form [model]="model" [fields]="fields" [options]="options" [form]="form2">
                             </formly-form>
+                            <!-- <app-forms form={{form}} identifier={{identifier}} modal="true"></app-forms> -->
                             <button type="button" data-dismiss="modal" aria-label="Close"
                                 class="btn btn-sec-bg">Cancel</button>
-                            <button type="submit" class="btn btn-primary float-end submit-button"
+                            <button type="submit" class="btn btn-primary float-end submit-button" id="dismissButton"
                                 [disabled]="!form2.valid">Save </button>
```

Comment:
@suvarnakale you are not doing anything using this id , remove it or set the proper button id name.